src/Book/CreateOrEditBook.tsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import "./CreateOrEditBook.css";
 import {Book, CREATE, Tag} from "./Book";
 import CreateAndCancelButtons from "../common/CreateAndCancelButtons/CreateAndCancelButtons";
+import ActivateDeactivateButton from "../common/ActivateDeactivateButton/ActivateDeactivateButton";
+import {toast, ToastContainer} from "react-toastify";
```

Comment:
What IDE are you using? (I recommend you [WebStorm](https://www.jetbrains.com/webstorm/))
I'm telling this because I saw some unused code or imports (like here) and if the code is not used is better to get rid of them.